src/components/molecules/BookMarkListItem.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import { IMAGES } from '../../constants/images'
+
+interface Props {
+  profileImage: string
+  title?: string
+  isMarked?: boolean
+}
+
+const BookMarkListItem: React.FC<Props> = (props: Props) => {
+  return (
+    <Container>
+      <Content>
+        <Image src={props.profileImage} />
+        <Title>{props.title}</Title>
+      </Content>
+      <BookMark
+        src={props.isMarked ? IMAGES.bookmark : IMAGES.bookmark_disabled}
+      />
+    </Container>
+  )
+}
+export default React.memo(BookMarkListItem)
+
+const Container = styled.div`
+  width: 343px;
```

Comment:
it would be better to reflect responsive width of container for both mobile and pc
i suggest percent value and min-width

Reply:
Good opinion! I'll test with pc and mobile And then I'll request review :)